fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformServiceImpl.java

```diff
@@ -264,11 +263,10 @@ public Collection<LoanTransactionData> retrieveLoanTransactions(final Long loanI
             final String sql = "select " + rm.loanPaymentsSchema() + " where tr.loan_id = ? and tr.transaction_type_enum not in (0, 3) "
                     + " and (tr.is_reversed=false or tr.manually_adjusted_or_reversed = true)  order by tr.transaction_date ASC,id ";
             Collection<LoanTransactionData> loanTransactionData = this.jdbcTemplate.query(sql, rm, loanId); // NOSONAR
+            // TODO: rework
```

Comment:
Is this something you forgot to do now and this was accidentally left here or the comment is for the future? If so, probably would be best if you explain it a little bit better whats the issue here - for future reference.

Reply:
nice catch. added a couple extra thoughts